dev/TOC.md

```diff
@@ -291,6 +291,7 @@
   + [TiDB Binlog 简介](reference/tidb-binlog-overview.md)
   + 生态工具
     - [tkctl](reference/tools/tkctl.md)
+    - [Kubernetes 上的工具使用](reference/tools/tools-in-kubernetes.md)
```

Comment:
should we move `tkctl` to `tools-in-kubernetes.md`?

Reply:
I suppose not, `tkctl` is the first-class **command line interface**, while this document describes the **related** tools